Gemfile.lock

```diff
@@ -292,8 +292,7 @@ GEM
     regexp_parser (2.9.2)
     reline (0.5.9)
       io-console (~> 0.5)
-    rexml (3.3.6)
-      strscan
+    rexml (3.3.8)
```

Comment:
Upgraded rexml to fix error caused by ruby upgrade, see [issue](https://github.com/ruby/rexml/pull/204#issuecomment-2327937133)

Reply:
Thank you!